Alloy/utils.js

```diff
@@ -521,6 +522,13 @@ exports.die = function(msg, e) {
 	process.exit(1);
 };
 
+exports.dieWithCodeFrame = function(msg, e, file) {
+	var rawLines = fs.readFileSync(file).toString();
+	var frame = codeFrame(rawLines, e.loc.line, e.loc.column, { highlightCode: true });
+	console.log(frame);
```

Comment:
I'm using plain old console.log here as if I use logger.info (or similar) the code frame gets shifted and looks bad